pilot/pkg/serviceregistry/kube/controller/autoserviceexportcontroller.go

```diff
@@ -126,9 +124,14 @@ func (c *autoServiceExportController) createServiceExportIfNotPresent(svc *v1.Se
 		},
 	}
 
-	serviceExports := c.client.MulticlusterV1alpha1().ServiceExports(svc.Namespace)
-	_, err := serviceExports.Create(context.TODO(), &serviceExport, metav1.CreateOptions{})
+	// Convert to unstructured.
+	u, err := runtime.DefaultUnstructuredConverter.ToUnstructured(&serviceExport)
```

Comment:
`serviceExport` here is still a `v1alpha1` struct. Can that cause issues?

Reply:
Good catch. Not sure, but added the APIVersion just in case.